hoomd/md/pytest/test_rigid.py

```diff
@@ -131,11 +137,22 @@ def check_orientation(
 
 
 @skip_rowan
+@pytest.mark.parametrize(
+    "create_bodies_args",
```

Comment:
These should be bools not dicts because only the test knows how many particles there are. It will help reduce duplication too.